apps/wallet/src/ui/app/components/menu/content/menu-list/index.tsx

```diff
@@ -56,6 +61,24 @@ function MenuList() {
                     subtitle={'v' + version}
                 />
             </div>
+            {WALLET_ENCRYPTION_ENABLED ? (
+                <div className={st.container}>
+                    <div className={st.item}>
+                        <Button
+                            mode="secondary"
+                            size="large"
+                            className={st.btn}
+                            onClick={async () => {
+                                await dispatch(lockWallet()).unwrap();
+                                navigate('/locked', { replace: true });
```

Comment:
Longer term we should probably just make this a side-effect of the locked state changing at a higher level, so that we never need to explicitly navigate to the locked page.

Reply:
yeah it is already there, this overrides the behaviour to navigate back to the menu when unlocking. (replaces /locked?url=... with just /locked) but yeah it can be inproved